Bio/SeqIO/__init__.py

```diff
@@ -420,6 +425,8 @@
     "gck": GckIO.GckIterator,
     "genbank": InsdcIO.GenBankIterator,
     "genbank-cds": InsdcIO.GenBankCdsFeatureIterator,
+    "gfa": GfaIO.GfaIterator,
```

Comment:
Assuming we can't easily detect the version from the file contents (which would be ideal, then we just need "gfa" as a single format), maybe "gfa1" and "gfa2" as format names?